.ci/snapshot.sh

```diff
@@ -35,3 +35,6 @@ fi
 
 echo "--- Deploy the snapshot :package: [./mvnw ${GOAL})] ${DRY_RUN_MSG}"
 ./mvnw -V -s .ci/settings.xml -Pgpg clean ${GOAL} -DskipTests --batch-mode | tee snapshot.txt
+
+echo "--- Archive the target folder with jar files"
+find . -type d -name target -exec find {} -name '*.jar' -print0 \; | xargs -0 tar -cvf "${TARBALL_FILE:-dist.tar}"
```

Comment:
Use `.ci/published-artifacts-list.sh` script added in https://github.com/elastic/ecs-logging-java/pull/251.

Reply:
see https://github.com/elastic/ecs-logging-java/pull/250/commits/332f334847f63edd084be2d83e17d696a8ad30ca